src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -124,13 +133,13 @@ public class BlobFileStore : IFileStore
         EnsureArg.IsNotNull(partition, nameof(partition));
         EnsureArg.IsNotNullOrWhiteSpace(blockId, nameof(blockId));
 
-        BlockBlobClient blobClient = GetInstanceBlockBlobClient(version, partition.Name);
-        _logger.LogInformation("Trying to read block list for DICOM instance file with watermark '{Version}'.", version);
+        BlockBlobClient blobClient = GetInstanceBlockBlobClient(version, partition, fileProperties);
+        _logger.LogInformation("Trying to read block list for DICOM instance file with path '{Path}'.", blobClient.Name);
```

Comment:
I missed my earlier comment about logging Path. The path can potentially contain sensitive information, why include that?